Cargo.lock

```diff
@@ -1,295 +1,294 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
I guess an update to the lockfile is due, but it should probably not be in the same PR